aiohttp/connector.py

```diff
@@ -1357,6 +1383,13 @@ def __init__(
         )
         self._path = path
 
+    @cached_property
+    def allowed_protocol_schema_set(self) -> frozenset[str]:
+        """Return allowed protocol schema set."""
+        return (
+            EMPTY_SCHEMA_SET | HTTP_SCHEMA_SET | WS_SCHEMA_SET | UNIX_PROTCOL_SCHEMA_SET
```

Comment:
Shouldn't UnixConnector only work with unix:?

Reply:
There are specific tests to allow http etc over unix in connector

Reply:
https://github.com/aio-libs/aiohttp/blob/c99a1e27375285149ea82cbdcc2f2c40e57596dc/tests/test_connector.py#L2419

https://github.com/aio-libs/aiohttp/blob/c99a1e27375285149ea82cbdcc2f2c40e57596dc/tests/test_connector.py#L2265